pkg/controller/database/cloudsql.go

```diff
@@ -248,3 +251,20 @@ func (t *cloudsqlTagger) Initialize(ctx context.Context, mg resource.Managed) er
 	}
 	return errors.Wrap(t.kube.Update(ctx, cr), errManagedUpdateFailed)
 }
+
+type specValidator struct{}
+
+// Initialize adds the external tags to spec.forProvider.settings.userLabels
```

Comment:
Looks like this may have been copied over from tags initializer

Reply:
Fixed 👍